### ISSUE TITLE
Mouse not working on the Android app

### ISSUE THREAD

**Issue by danZ4C** [veloce/lichobile#1982]
It seems that the lichess android app cannot be operated with a mouse.

I often use my mobile / tablet with bluetooth mouse, both directly on the screen or an external monitor, and whilst the mouse (coursor) works perfectly fine with with every app on my phone (apart from some elements where clicking and dragging the pointer acts as text selection instead of as a finger gesture/movement/drag - which is perfectly logical).

But the lichess app totally ignores the mouse, clicking doesnt do anything enywhere within the app.

That on samsung galaxy s8, and I have noticed that Lichess app is seen by the system as a game, and it enters "game booster" mode, but i think ive disabled any of those features, and non of the settings mention anything about mice.

**Comment by jas14** [comment]
I don't believe veloce is planning to support mouse usage in the app, but in any case, this is a duplicate of #1239
